articles/libraries/lock/index.md

```diff
@@ -150,38 +114,25 @@ ${lockSDK}
 
 Browser compatibility is ensured for **Chrome**, **Safari**, **Firefox** and **IE >= 9**. Auth0 currently uses [zuul](https://github.com/defunctzombie/zuul) along with [Saucelabs](https://saucelabs.com) to run integration tests on each push.
 
-## Issue Reporting
-
-If you find a bug or have a feature request, report them in Github at [Auth0 Lock Issues](https://github.com/auth0/lock/issues). However, do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
-
-## Additional Resources
-
-* [Lock 9 to Lock 10 Migration Guide][migration-guide]
-* [Complete API][lock-api]
-* [UI Customization][ui-customization]
-* [Customizing Lock to Your Needs][lock-customization]
-* [Embedding Lock (Instead of a Modal)][display-modes]
-* [i18n - Languages and Custom Text][i18n-notes]
-* [Popup Mode][popup-mode] - an advanced option (not recommended in most cases)
-* [Development notes][development-notes]
-* [Release process notes][release-process]
-* [Migration guide][legacy-migration-guide] for users of the legacy **Auth0 Widget**
-
-<!-- Variables-->
+<!--vars-->
```

Comment:
@jeffreylees Also (again out of curiosity), why have these vars here as well, since they already are at the `_lock-toc.md` file?